assets/src/edit-story/utils/getInUseFonts.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function getInUseFontsForPages(pages) {
```

Comment:
This function enumerates through all the pages and creates a unique array of font names.

Reply:
On the surface there might be some overlap with `assets/src/edit-story/output/utils/getFontDeclarations.js`. Perhaps there's something that could be shared between the two, or at least referenced in the jsdoc (which are currently missing). Anyhow, just dropping this here

Reply:
Sounds good. I will add some JSDocs. There is some overlap, but we don't need the in-depth nature of getting weights or building the string. The only really in common line is:

`const textElements = elements.filter(({ type }) => type === 'text');`

which is probably not worth extracting out, but I don't feel strongly either way.